app/2.3.x/hybrid-mode.md

```diff
@@ -369,6 +380,39 @@ Once the nodes are setup, use the Admin API on the Control Plane as usual,
 those changes will be synced and updated on the Data Plane nodes
 automatically within seconds.
 
+## Version and compatibility checks
+
+Since Kong release 2.3.0, a new feature has been added which allows
+Control Plane nodes to determine if the Data Plane connected can be
```

Comment:
```suggestion
Starting with Kong Gateway 2.3.0, Control Plane nodes can perform version 
compatibility checks to determine if connected Data Planes can be
```
Avoid saying "a new feature", since I doubt anyone will be updating this sentence in upcoming releases.